### ISSUE TITLE
Scrollbars are too thin

### ISSUE THREAD

**Issue by joshas** [tutao/tutanota#10849]
### Pre-checks

- [x] This is not a feature request. Existing functionality does not work as intended, **not** missing functionality. I will request features via [discussions](https://github.com/tutao/tutanota/discussions/), [reddit](https://www.reddit.com/r/tutanota/) or [support](https://tuta.com/support)
- [x] I've searched and did not find a similar issue.
- [x] My report does not have personal or private information like email address or recovery code

### App version

v348.260519.0

### Browser

Firefox 

### Browser version

151.0 (64-bit)

### Operating System

Linux

### What happened?

Accessibility issue: scrollbars are too thin and cannot be easily used with mouse. To improve accessibility there should be at least an option to switch scrollbars to default width. Current width is "thin", and it is only 3px wide.

**Comment by charlag** [comment]
Hi, thank you for reporting it.

As a web app we only have a choice between thinner and thicker scrollbars, the rest is up to the browser and OS. Firefox has a hidden setting called `layout.css.scrollbar-width-thin.disabled` which they should ideally expose via settings.

We will consider what to do about this issue, thank you.

**Comment by joshas** [comment]
I did not know about `layout.css.scrollbar-width-thin.disabled` setting in Firefox. Thank you, this fixes so many issues in other websites too :)
